packages/data-service-generator/tests/e2e/plugins.ts

```diff
@@ -36,13 +36,13 @@ export const basicAuth: PluginInstallation[] = [
     pluginId: "auth-core",
     npm: "@amplication/plugin-auth-core",
     enabled: true,
-    version: "1.1.0",
+    version: "2.0.0-beta.1",
```

Comment:
We should add a task to remind us to change this one to "latest" as well after we publish the new auth-core.

Reply:
I'll raise a PR straight after we merge this that will update with the v2.0.0.
I'll release the plugin v2.0.0 tomorrow with the beta tag